avaframe/com4FlowPy/splitAndMerge.py

```diff
@@ -176,3 +176,34 @@ def MergeRaster(inDirPath, fName):
             log.info("appended result %s_%i_%i", fName, i, j)
 
     return mergedRas
+
+def MergeRaster_min(inDirPath, fName):
```

Comment:
Please use our convention of mergeRasterMin. And I suggest to start adding docstrings now